odata-temporal-ext/3 Vocabulary.md

```diff
@@ -0,0 +1,133 @@
+-------
+
+# ##sec Vocabulary for Temporal Data
+
+The vocabulary for temporal data [OData-VocTemporal](#ODataVocTemporal)
+defines a structured term for describing temporal query and modification
+capabilities of temporal services.
+
+The term `ApplicationTimeSupport` describes the temporal capabilities of
+an entity set. It can be applied to an entity set, or to a containment
+navigation property to describe its implicit entity sets. It has a
+structured type with the following properties:
+
+- `UnitOfTime` describes the data type of
+  the application-time period start and end values. Its value is a record
+  of either type `UnitOfTimeDate` or `UnitOfTimeDateTimeOffset`.
+  - Records of type `UnitOfTimeDateTimeOffset` MUST specify the property `Precision`.
+  - Records of type `UnitOfTimeDate` MAY specify the property `ClosedClosedPeriods`.
+- `Timeline` describes the kind of
+  temporal entity set. Its value is a record of either type
+  `TimelineSnapshot` or `TimelineVisible`.
+  - Records of type `TimelineSnapshot` MUST NOT specify any properties.
+  - Records of type `TimelineVisible` MUST specify the properties `PeriodStart` and `PeriodEnd`.
+    - These properties MUST point to
+      properties of the annotated entity set whose values are the period start
+      and end of a time slice.
+    - The start is always included in the
+      time slice, the end is included if and only if the sibling property
+      `UnitOfTime` is of type `UnitOfTimeDate` and specifies property
+      `ClosedClosedPeriods` as `true`.
+    - If the period end property does not
+      specify a default value, a default value of "ad infinitum" is assumed.
+  - Records of type `TimelineVisible` MAY specify the property `ObjectKey`.
+    - `ObjectKey` is the “sub-key” or “alternate key” that identifies time slices for a single temporal object. 
+      It is only necessary if the annotated entity set can contain time slices
+      for more than one temporal object. `The object key is `a collection of
+      property paths whose value combination uniquely identifies a temporal
+      object.
+- `SupportedActions` is a collection of
+  qualified names of temporal actions that may be bound to the annotated
+  entity set. These can be the [standard actions defined in this specification](#ModifyingTemporalData),
+  or service-specific actions.
+
+::: example
+Example ##ex: `Employees` entity set from [example model `api-1`](#api1)
+annotated with temporal terms
+```json
+"Employees": {
+  "$Collection": true,
+  "$Type": "OrgModel.Employee",
+  "@Temporal.ApplicationTimeSupport": {
+    "Timeline": {
+      "@odata.type": "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Temporal.V1.json#Temporal.TimelineSnapshot"
```

Comment:
Break these overlong lines

Reply:
How/why? Breaking within the string would make it invalid JSON, and just moving the overlong string into a separate line wouldn't help much.

Reply:
```suggestion
      "@odata.type": "https://oasis-tcs.github.io/odata-vocabularies/
        vocabularies/Org.OData.Temporal.V1.json#Temporal.TimelineSnapshot"
```
looks better than

![image](https://github.com/oasis-tcs/odata-specs/assets/67904016/2c7679ce-3794-481f-b282-c80667d17efe)

Reply:
I feel uncomfortable writing down incorrect JSON just for better looks.